app/lib/common/widgets/avatar/full_screen_avatar_page.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:acter/common/providers/room_providers.dart';
+import 'package:acter/common/utils/utils.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+import 'package:zoom_hover_pinch_image/zoom_hover_pinch_image.dart';
+
+class FullScreenAvatarPage extends ConsumerWidget {
+  final String roomId;
+
+  const FullScreenAvatarPage({super.key, required this.roomId});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Scaffold(
+      appBar: _buildAppBar(context, ref),
+      body: _buildBody(context, ref),
+    );
+  }
+
+  AppBar _buildAppBar(BuildContext context, WidgetRef ref) {
+    final membership = ref.watch(roomMembershipProvider(roomId)).requireValue;
```

Comment:
```suggestion
    final membership = ref.watch(roomMembershipProvider(roomId)).valueOrNull;
```

I feel like you should create a custom rewrite in your editor that replace `requireValue` with `valueOrNull` on typing ;) .

Reply:
Oh no, Again!! :(